core/storage/suggestion/gae_models.py

```diff
@@ -479,17 +480,29 @@ def get_in_review_translation_suggestions(
                 As a user cannot review their own suggestions, suggestions
                 authored by the user will be excluded.
             language_codes: list(str). The list of language codes.
+            exp_ids: list(str). Exploration IDs matching the target ID of the
```

Comment:
I think this needs more documentation to explain what is returned in the exp_ids = [] case.

Also, on further thought, it seems odd to combine the two -- you end up with this weird branching. I suggest making two separate methods, instead.

Reply:
Yes, I discussed with Sagang earlier about going ahead with the if-else conditional logic, but now that we have another case to handle as pointed by Jim, I have created a seperate function altogether.